docs/lesiaturuta/PokemonProject/style.css

```diff
@@ -0,0 +1,776 @@
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+}
+
+.calculate__result {
+  height: 36px;
+}
+
+.calculate__first-value,
+.calculate__operation,
+.calculate__second-value {
+  color: #221F1F;
+}
+
+.calculate__button {
+  background: #EF4934;
+  color: #FFF;
+  border: none;
+}
+
+.calculate__result {
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 36px;
+  color: #221F1F;
+}
+
+.calculate__show {
+  width: 100%;
+  margin-top: 80px;
+  display: flex;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 48px;
+  color: #EF4934;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.calculate__show-operation,
+.calculate__show-equals {
+  margin: 0 100px;
+}
+
+.main__calculate {
+  align-items: flex-start;
+}
+
+/*ERRORS*/
+
+.error-text {
+  position: absolute;
+  bottom: -20px;
+  left: 0;
+  color: #FA0A0A;
+}
+
+.border-error {
+  border: 4px solid #FA0A0A;
+}
+
+.border-ok {
+  border: 4px solid #66EA11;
+}
+
+/*HELPER*/
+
+.height100vh {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.text-weight {
+  font-weight: 600;
+  line-height: 21px;
+  margin-right: 10px;
+}
+
+.flex-direction-column {
+  flex-direction: column;
+}
+
+.gap-top6 {
+  margin-top: 6px;
+}
+
+.gap-top12 {
+  margin-top: 12px;
+}
+
+.gap-right6 {
+  margin-right: 6px;
+}
+
+.gap-right33 {
+  margin-right: 33px;
+}
+
+.color-background-FDB714 {
+  background-color: #FDB714;
+}
+
+.color-background-EF4934 {
+  background: #EF4934;
+}
+
+.color-background-FF87F3 {
+  background: #FF87F3;
+}
+
+.color-background-3447EF {
+  background: #3447EF;
+}
+
+.flex-direction-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.display-none {
+  display: none;
+}
+
+@media screen and (max-width: 1220px) {
+
+  .nav__container {
+    padding: 0 10px;
+  }
+
+  .main__container {
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .cards {
+    grid-template: repeat(2, 1fr)/repeat(3, 1fr);
+    gap: 10px;
+    justify-items: center
+  }
+
+  .form__search {
+    padding: 0 10px;
+  }
+}
+
+@media screen and (max-width: 630px) {
+
+  .nav {
+    height: 100px;
+  }
+
+  .nav__menu {
+    margin-top: 10px;
+  }
+
+  .nav__container {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .login__form-button {
+    padding: 0 50px;
+  }
+}
+
+@media screen and (max-width: 1000px) {
+
+  .form__search {
+    height: 200px;
+  }
+
+  .form__search-container {
+    flex-direction: column;
+  }
+
+  .form__search-select,
+  .form__search-radio,
+  .form__search-checkbox,
+  .form__search-input {
+    margin-top: 15px;
+  }
+}
+
+@media screen and (max-width: 880px) {
+
+  .cards {
+    grid-template: repeat(2, 1fr)/repeat(2, 1fr);
+  }
+}
+
+@media screen and (max-width: 600px) {
+
+  .cards {
+    grid-template: repeat(2, 1fr)/repeat(1, 1fr);
+  }
+
+  .title-pokemon {
+    font-size: 30px;
+    text-align: center;
+  }
+}
+
+@media screen and (max-width: 480px) {
+
+  .nav {
+    height: 190px;
```

Comment:
Please try to avoid fixed width and height. You can use `min-width`/ `max-width` and `min-height`/`max-height` instead

Reply:
fixed